tests/infra/remote.py

```diff
@@ -135,6 +150,13 @@ def __init__(
         self.root = os.path.join(workspace, label + "_" + name)
         self.name = name
         self.env = env or {}
+        self.out = os.path.join(self.root, "out")
+        self.err = os.path.join(self.root, "err")
+        self.analytics_out = None
```

Comment:
`self.analytics_out = os.path.join(log_path, f"out_{label}_{name}") if log_path else None` is probably more pythonic.